NGINX/nginx/ingest/parser.yml

```diff
@@ -0,0 +1,180 @@
+          network.protocol: '{{parsed_event.message.http_scheme}}'
+          source.address: '{{parsed_event.message.source_address}}'
+          source.ip: '{{parsed_event.message.remote_addr or parsed_event.message.source_address}}'
+          source.user.name: '{{parsed_event.message.source_user_name}}'
+          url.domain: '{{parsed_event.message.url_domain or parsed_event.message.url_name}}'
+          url.original: >
+            {%- if parsed_event.message.get("url_original") != None -%}
+              {{parsed_event.message.url_original}}
+            {%- elif parsed_event.message.get("request") != None -%}
+              {%- set list = parsed_event.message.request.split(' ') -%}
+              {{list[1]}}
+            {%- endif -%}
+          url.path: '{{parsed_event.message.url_path}}'
+          url.port: '{{parsed_event.message.url_port}}'
+          url.scheme: '{{parsed_event.message.url_scheme}}'
+          url.username: '{{parsed_event.message.url_username}}'
+          user_agent.original: '{{parsed_event.message.http_user_agent}}'
+      - set:
+          http.request.referrer: '{{parsed_event.message.http_referrer or parsed_event.message.http_referer}}'
+        filter: '{{parsed_event.message.get("http_referrer") != "-" and parsed_event.message.get("http_referer") != "-"}}'
+      - set:
+          action.name: >
+            {%- if parsed_event.message.get("request_method") != None -%}
+              {{parsed_event.message.request_method}}
+            {%- elif parsed_event.message.get("request") != None -%}
+              {%- set list = parsed_event.message.request.split(' ') -%}
+              {{list[0]}}
+            {%- endif -%}
+          action.outcome: >
+            {%- if parsed_event.message.get("response_status") != None -%}
+              {%- if parsed_event.message.response_status < 200 -%}
+                information
+              {%- elif parsed_event.message.response_status < 300 -%}
+                success
+              {%- elif parsed_event.message.response_status < 400 -%}
+                redirect
+              {%- else -%}
+                failure
+              {%- endif -%}
+            {%- elif parsed_event.message.get("status") != None -%}
+              {%- if parsed_event.message.status | re_search("^([0-9]|[0-9][0-9]|1[0-9][0-9])$") -%}
+                information
+              {%- elif parsed_event.message.status | re_search("^2[0-9][0-9]$") -%}
+                success
+              {%- elif parsed_event.message.status | re_search("^3[0-9][0-9]$") -%}
+                redirect
+              {%- else -%}
+                failure
+              {%- endif -%}
+            {%- endif -%}
+        filter: '{{original.message | re_match("\{(?:[^{}]|())*\}")}}'
+      - translate:
+        dictionary:
+          'GET': ['access']
+          'DELETE': ['deletion']
+        mapping:
+          action.name: event.type
```

Comment:
Event with category `web` only accept type `access`, `error` or `info` (cf [doc](https://www.elastic.co/guide/en/ecs/8.6/ecs-allowed-values-event-category.html#ecs-event-category-web)).
I suggest deleting this step as we have the step line 170 that will set the `event.type`